configs/trainer.yaml

```diff
@@ -10,3 +10,4 @@ wandb:
   entity: turing-arc
   project: ms2
   mode: online
+  id_postfix: _test_run #this postfix MUST be changed on each run
```

Comment:
Following our discussion yesterday, I've added the utility functions to ensure that a given run name cannot exist prior to a model being trained, and must be unique prior to transfer attacks being done. This will enforce uniqueness on the user. I'll make sure to add to README